recipes/blimpy/meta.yaml

```diff
@@ -0,0 +1,83 @@
+{% set name = "blimpy" %}
+{% set version = "2.0.40" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/blimpy-{{ version }}.tar.gz
+  sha256: 29c0e3715399cc46895cde6a9d6f3f75b348001b8855139fac90c8b0ee1e860a
+
+build:
+  entry_points:
+    - watutil = blimpy.waterfall:cmd_tool
+    - rawutil = blimpy.guppi:cmd_tool
+    - fil2h5 = blimpy.fil2h5:cmd_tool
+    - h52fil = blimpy.h52fil:cmd_tool
+    - h5diag = blimpy.h5diag:cmd_tool
+    - bl_scrunch = blimpy.bl_scrunch:cmd_tool
+    - matchfils = blimpy.match_fils:cmd_tool
+    - bldice = blimpy.dice:cmd_tool
+    - calcload = blimpy.calcload:cmd_tool
+    - rawhdr = blimpy.rawhdr:cmd_tool
+    - stax = blimpy.stax:cmd_tool
+    - stix = blimpy.stix:cmd_tool
+    - peek = blimpy.peek:cmd_tool
+    - srcname = blimpy.srcname:cmd_tool
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - pytest-runner
+    - python
+  run:
+    - astropy
+    - cython
```

Comment:
Maybe open an issue upstream because it looks like cython is unused in the current version.

Reply:
I opened an issue upstream to address this https://github.com/UCBerkeleySETI/blimpy/pull/265